src/showcase/Showcase.story.mdx

```diff
@@ -0,0 +1,34 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+
+import { Paragraph } from '../paragraph'
+import { Showcase } from './index'
+
+<Meta title="Widgets/Showcase" />
+
+# Showcase
+
+<Canvas>
+  <Story name="Default">
+    <Showcase>
+      <div>Lists and loops</div>
```

Comment:
Do we need this very wide HTML-based API? Can we just have a list of stings ?

Reply:
We are presented with two use cases, one that slides between a new custom component that will include an itinerary, and another that slides between subheader components. So I figured I would not restrict what you can pass to the Showcase/Marquee component.

Great thinking on pausing the animation on hover to read it. It's not in the specs (well there's not much specs anyway) but I'll add it.